src/rcc/mod.rs

```diff
@@ -485,6 +490,49 @@ impl Rcc {
             },
         };
 
+        // See RM0455 Rev 3 Table 15
+        #[cfg(feature = "rm0455")]
+        let (wait_states, progr_delay) = match vos {
+            // VOS 0 range VCORE 1.25V - 1.35V
+            Voltage::Scale0 => match rcc_aclk_mhz {
+                0..=41 => (1, 0), // Errata? 7A3 Rev Z did not run at 0 wait-states
```

Comment:
Might be worth to investigate further before we merge this?

Reply:
Honestly for me this doesn't matter. It would only be important to squeeze out the last bit of performance at low clock speeds. The next step would be to check if ST's example code ever tries setting zero-wait states, and if it does something different.